plugins/poll/assets/javascripts/discourse/components/poll.gjs

```diff
@@ -253,7 +253,7 @@ export default class PollComponent extends Component {
   }
 
   get voters() {
-    return this.poll.voters;
+    return this.poll.get("voters");
```

Comment:
For my own benefit, why is this change necessary - it was working fine?

Reply:
it was not working fine in my tests :)